### PR TITLE
Add option to configure keep-alive for TCP connections

### LINKED ISSUE #13: Add keepalive
Please add keepalive to TCP transport:
GelfTcpTransport
```java
        bootstrap.group(workerGroup)
                .channel(NioSocketChannel.class)
                .option(ChannelOption.CONNECT_TIMEOUT_MILLIS, config.getConnectTimeout())
                .option(ChannelOption.TCP_NODELAY, config.isTcpNoDelay())
                .option(ChannelOption.SO_KEEPALIVE, config.isKeepAlive())
                .option(ChannelOption.SO_TIMEOUT, config.getConnectTimeout())
```

### PR DESCRIPTION
**Pull request by bernd**
The default is false to keep previous behaviour.

Fixes #13

### CODE REVIEW

File: src/test/java/org/graylog2/gelfclient/GelfConfigurationTest.java [hunk 1]
```diff
@@ -167,6 +167,16 @@ public void testtcpNoDelay() {
     }
 
     @Test
+    public void testtcpKeepAlive() {
+        // Check default value.
+        assertEquals(false, config.isTcpKeepAlive());
+
+        config.tcpKeepAlive(true);
+
+        assertEquals(true, config.isTcpKeepAlive());
```
Comment (joschi) [inline src/test/java/org/graylog2/gelfclient/GelfConfigurationTest.java]: `assertTrue` maybe? :wink:

**Comment by bernd** [comment]
Updated after comment.

File: src/test/java/org/graylog2/gelfclient/GelfConfigurationTest.java (hunk 1 above)
Comment (bernd) [inline src/test/java/org/graylog2/gelfclient/GelfConfigurationTest.java]: Ja :grinning:

Comment (bernd) [inline src/test/java/org/graylog2/gelfclient/GelfConfigurationTest.java]: Args, missed some.

**Comment by bernd** [comment]
More fixes.

**Comment by joschi** [comment]
LGTM. :+1:

**Pull request merged**